packages/ui/test/proposals/components/ProposalListItem.test.tsx

```diff
@@ -119,7 +119,7 @@ describe('UI: ProposalListItem', () => {
         server.server
       )
 
-      renderComponent({ proposal: { ...proposalData, status: 'dormant' }, memberId: '0' })
+      renderComponent({ proposal: { ...proposalData, status: 'dormant' }, isCouncilMember: true, memberId: '0' })
```

Comment:
Why is this needed ?

Reply:
Because I updated `ProposalListItem.tsx`.
So it is needed to pass the test.

Reply:
Oh right! I had not realized that the `ProposalItemVoteDetails` was always shown before, and always queried `useProposalVotesByMember` for all users once per proposal (this was wasteful !)

Since it's not the case anymore could you remove `isCouncilMember` from `ProposalItemVoteDetailsProps` please ?

But more importantly has you can see on [this story](https://pioneer-2-storybook-git-fork-eshark9312-rwd-pr-3a2165-joystream.vercel.app/?path=/story/pages-proposals-proposallist-current--default&args=isCouncilMember:!true) (you might need to set `isCouncilMember` to `true` manually to see it) the "my vote" header is broken for council members:

![image](https://github.com/Joystream/pioneer/assets/6571453/88faad07-6727-4c2b-8817-2686a6e5f828)

Reply:
I got it.
Updated the code.